notebooks/official/automl/automl_image_classification_online_prediction.ipynb

```diff
@@ -645,7 +586,7 @@
       "source": [
         "### Get test item\n",
         "\n",
-        "You will use an arbitrary example out of the dataset as a test item. Don't be concerned that the example was likely used in training the model -- we just want to demonstrate how to make a prediction."
+        "You use an arbitrary example out of the dataset as a test item. Don't be concerned that the example was likely used in training the model -- we just want to demonstrate how to make a prediction."
```

Comment:
Change from: 
- "we just want to demonstrate how to make a prediction"
- To:
    - "the point is to demonstrate how to make a prediction"